compiler/src/iree/compiler/Codegen/LLVMGPU/Utils/LLVMGPUSelectUKernels.cpp

```diff
@@ -58,24 +63,38 @@ static UKernelNameAndSuffix getUKernelNameAndSuffix(Operation *op) {
   return {};
 }
 
-// Returns the UKernelConfigAttr for any op. Returns {} if no ukernel.
-static IREE::GPU::UKernelConfigAttr getUKernelConfig(Operation *op) {
+static int64_t getSharedMemorySize(IREE::GPU::TargetAttr gpuTarget) {
```

Comment:
`getSharedMemorySizeInBytes` or `getSharedMemoryBytes`? I think it's important to be explicit here since the allocation unit of shared memory is dwords.

Reply:
Good idea (actually I've been back and forth on this. I didn't know about dwords being the standard unit for shared memory).